src/list.json

```diff
@@ -151,7 +151,16 @@
         "link": "https://blog.tooljet.com/hacktoberfest-2022/"
       }
     ],
-    "U": [],
+    "U": [
```

Comment:
hey @sshiv5768 you need to Re-Style it via running this command in the repo directory
`git checkout <your branch> `
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2336040/patch | git am`
`git push`

Reply:
Ok Will do it.

Reply:
@thefool76 check new commits